### PR TITLE
fix: add combined stops/stations layer

### PR DESCRIPTION
**Pull request by miles-grant-ibigroup**
Requires a few hacks to get click handling working, but allows us to render stations/stops seamless as one layer

### CODE REVIEW

File: packages/otp2-tile-overlay/src/index.tsx [hunk 1]
```diff
@@ -224,14 +254,19 @@ const OTP2TileLayerWithPopup = ({
...
-  layers: { color?: string; name?: string; network?: string; type: string, initiallyVisible?: boolean }[],
+  layers: { color?: string; name?: string; network?: string; type: string, initiallyVisible?: boolean, overrideType?: string }[],
   endpoint: string,
   setLocation?: (location: MapLocationActionArg) => void,
   setViewedStop?: (stop: Stop) => void,
   stopsWhitelist?: string[],
   configCompanies?: ConfiguredCompany[],
   getEntityPrefix?: (entity: Stop | Station) => JSX.Element
 ): JSX.Element[] => {
+  const fakeOtpUiLayerIndex = layers.findIndex(l=>l.type === STOPS_AND_STATIONS_TYPE)
```
Comment (amy-corson-ibigroup) [inline packages/otp2-tile-overlay/src/index.tsx:265]: Can you add a comment here explaining why we're calling this `fakeOtpUiLayerIndex`

**Review by amy-corson-ibigroup — CHANGES_REQUESTED** [review]
Only issue I'm seeing is this is not playing well with the entity prefix. Are you anticipating fixing that in OTP-RR or is there a way to not pass it to stations?
image

File: packages/otp2-tile-overlay/src/index.tsx (hunk 1 above)
Comment (miles-grant-ibigroup) [reply packages/otp2-tile-overlay/src/index.tsx:265]: It's not a real layer! The others are all linked to actual maplibre layers

**Comment by miles-grant-ibigroup** [comment]
Bug you mention is fixed here! https://github.com/opentripplanner/otp-react-redux/pull/1284

**Review by amy-corson-ibigroup — APPROVED** [review]
OTP-RR draft looks great thanks for the explanation!

**Review by josh-willis-arcadis — APPROVED** [review]
looks great!

**Pull request merged**